examples/image_classifier/alexnet/README.md

```diff
@@ -1,4 +1,4 @@
-#### Sample commands to create a alexnet eager mode model archive, register it on TorchServe and run image prediction
+### Sample commands to create a alexnet eager mode model archive, register it on TorchServe and run image prediction
```

Comment:
Since we show both compile & eager,
```suggestion
### Sample commands to create a alexnet model archive, register it on TorchServe and run image prediction
```